src/phylum/ci/ci_base.py

```diff
@@ -179,7 +193,14 @@ def _check_prerequisites(self) -> None:
         if self.phylum_project_file.exists():
             print(f" [+] Existing `.phylum_project` file found at: {self.phylum_project_file}")
         else:
-            raise SystemExit(" [!] The `.phylum_project` file was not found at the current working directory")
+            if self.phylum_project:
+                print(f" [+] No existing `.phylum_project` file found at: {self.phylum_project_file}")
+                print(f" [+] A Phylum project will be created with the provided name: {self.phylum_project}")
```

Comment:
The language on this could be improved. Users should know that they can also use this with existing projects.

```suggestion
                print(f" [+] A Phylum project will be used (or created) with the provided name: {self.phylum_project}")
```